packages/webpack/src/runtime/runtime-namespace.ts

```diff
@@ -1,6 +1,10 @@
 import { EndowmentsToolkitFactory, LavaMoatPolicy } from 'lavamoat-core'
 import { LavaMoatPluginOptions } from '../plugin'
 
+type DebugTools = {
```

Comment:
seems like this should be exported if `runtime.js` is using it?

Reply:
It seemed to propagate without that. I'll doublecheck.